src/Datadog.Trace/Vendors/Serilog.Sinks.File/Sinks/File/PathRoller.cs

```diff
@@ -56,7 +56,8 @@ public PathRoller(string path, RollingInterval interval)
                 "(?<" + PeriodMatchGroup + ">\\d{" + _periodFormat.Length + "})" +
                 "(?<" + SequenceNumberMatchGroup + ">_[0-9]{3,}){0,1}" +
                 Regex.Escape(_filenameSuffix) +
-                "$");
+                "$",
+                RegexOptions.Compiled);
```

Comment:
Slower construction, faster matching. I wonder if this will be noticeable in the relenv?